### ISSUE TITLE
DocumentationMethod

### ISSUE THREAD

**Issue by IngoTomahogh** [rubocop-hq/rubocop#6673]
### Expected behavior
The Style/DocumentationMethod cop should recognize documention comments even with an inlined `module_function` modifier.
### Actual behavior
The comment preceding a method with an inline `module_function` modifier is not recognized as the documentation for that method.
### Steps to reproduce the problem
```ruby
module Use
  # This is a useful function
  module_function def ful
  end
end
```
Rubocop will generate a `Style/DocumentationMethod: Missing method documentation comment.` message.
### Rubocop version
```
$ rubocop -V
0.62.0 (using Parser 2.5.3.0, running on ruby 2.5.1 x86_64-darwin17)
```
